src/Common/LargeBtn.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import PropTypes from 'prop-types';
+import styles from './themeStyle';
+import themeColor from './Color';
+
+export default function LargeButton(props) {
+  const { subtitle } = props;
+  const { title } = props;
+  const { note } = props;
+  const { bgColor } = props;
+  const btnStyle = `${bgColor}LargeBtn`;
+  const { btnAmount } = props;
+  const btnWidth = `${100 / btnAmount - 2}%`;
```

Comment:
LargeButton name suggests a generic component however it deals with `amount` so it is more specific. Lets move it to the stats screen and find another name

Reply:
Update the name into NumOfBtn

Reply:
Do not use that many abbreviation, let's call it `numberOfBtns` @KatyTao

Reply:
Done